### PR TITLE
Added a note without requesting into github team

### PR DESCRIPTION
**Pull request by himorin**

### CODE REVIEW

File: services/usage.rst [hunk 1]
```diff
@@ -310,9 +310,10 @@ GitHub
 PFS project uses `Subaru-PFS organization at GitHub <https://github.com/Subaru-PFS>`_. 
 In 'Subaru-PFS' organization at GitHub, 
 there is one (mostly private) `team per institute <https://github.com/orgs/Subaru-PFS/teams>`_
```
Comment (yukimoritani) [inline services/usage.rst]: > there  is one (mostly private) team per institute

LAM and PU have two teams for ICS and DRP separately.
how about changing as follows?

> there is basically one (mostly private) team per institute and development package, where...

File: services/usage.rst [hunk 2]
```diff
@@ -310,9 +310,10 @@ GitHub
...
-where the members have the admin privilege.
-You will need to be in one team of this organization 
-for push access to repositories. 
+where the members have the admin privilege to repositories owned.
+You will need to be in this organization (by invitation from the organization),
```
Comment (yukimoritani) [inline services/usage.rst]: I think you can leave the old words to explain more clearly what you mean by admin privilege..
For instance,
> To push repositories in our organization, you need ...

And , just for clarification, you can change   "this" to "Subaru-PFS" .

File: services/usage.rst [hunk 3]
```diff
@@ -310,9 +310,10 @@ GitHub
 PFS project uses `Subaru-PFS organization at GitHub <https://github.com/Subaru-PFS>`_. 
 In 'Subaru-PFS' organization at GitHub, 
 there is one (mostly private) `team per institute <https://github.com/orgs/Subaru-PFS/teams>`_
-where the members have the admin privilege.
-You will need to be in one team of this organization 
-for push access to repositories. 
+where the members have the admin privilege to repositories owned.
+You will need to be in this organization (by invitation from the organization), 
+and need to be in team(s) of this organization for admin privilege to
```
Comment (yukimoritani) [inline services/usage.rst]: privilege to -> privilege of

**Review by yukimoritani — COMMENTED** [review]
I have commented hoping to have better clarify.
I hope they will work you somehow.

File: services/usage.rst (hunk 1 above)
Comment (himorin) [reply services/usage.rst]: added.

File: services/usage.rst (hunk 2 above)
Comment (himorin) [reply services/usage.rst]: hrm. considering 'member of a team' is not a requirement, but 'member of Subaru-PFS organization' is a requirement, it seems this section need to be modified more to make clear...
how about newly pushed one?

**Comment by himorin** [comment]
For outside collaborator, it might be better to comment that permission is per repository but not to the entire organization, with adding a link to help page:
https://help.github.com/articles/adding-outside-collaborators-to-repositories-in-your-organization/

**Comment by yukimoritani** [comment]
I'm afraid you could put the link where "outside collaborator" shows up for the first time.
Ah... I apologize for not  having changed the view...
I'll do later, seeing whether you change the link or not.

**Review by yukimoritani — APPROVED** [review]
I don't have any more comments.

**Pull request merged**